riscv_isac/plugins/internaldecoder.py

```diff
@@ -341,6 +342,197 @@ def arithi_ops(self, instrObj):
 
         return instrObj
 
+    # Automatically generated by parse_opcodes.
+
+    # Put the following function in internaldecoder.py 
+    def rvp_ops(self, instrObj):

```

Comment:
The `rvp` specific dictionaries can probably be defined as a part of the decoder class over here.

Reply:
Done in 571c5525724b34f23a346e97628a1b1bc144be59